src/src/WebServer/HardwarePage.cpp

```diff
@@ -50,6 +51,14 @@ void handle_hardware() {
 #endif
     #ifdef ESP32
       Settings.InitSPI                = getFormItemInt(F("initspi"), 0);
+      if (Settings.InitSPI == 3) { // User-define SPI GPIO pins
```

Comment:
I see these comments about the `InitSPI` values, so why not make them defines? (or an enum class)

Reply:
Ok, I'll improve that.